executor/runtime/docker/docker.go

```diff
@@ -165,10 +166,21 @@ func NewConfig() (*Config, []cli.Flag) {
 				"systems. Kernels with CONFIG_RT_GROUP_SCHED=y require all cgroups in the hierarchy to have some " +
 				"cpu.rt_runtime_us allocated to each one of them",
 		},
+		cli.BoolTFlag{
+			Name:        "titus.executor.enableDefaultTitusTags",
+			Destination: &cfg.enableDefaultTitusTags,
+			Usage:       "When enabled, titus-executor Atlas metrics are decorated with additional tags, t.jobId, t.taskId",
+		},
 	}
 	return cfg, flags
 }
 
+// IsEnableDefaultTitusTags returns true if Atlas metrics should be tagged with additional tags
+// including t.jobId and t.taskId; return false otherwise
+func (c *Config) IsEnableDefaultTitusTags() bool {
```

Comment:
This should really be in its own config. You can just make this a flag inside of the main.go for the titus-executor, and then submit a true / false to the NewRunner function.